databases/week2/task1 and 3.sql

```diff
@@ -0,0 +1,22 @@
+--task1
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id.
+INSERT INTO task (title,description,created,updated,due_date,status_id,user_id) VALUES ("Adding","Adding new row", "2021-09-22 12:06:06", "2021-09-27 18:06:06","2021-09-30 12:00:00",1, 2);
+--•	Change the title of a task.
+UPDATE task SET title = "Did HYF Homework"WHERE id = 10;
+--•	Change a task due date
+UPDATE task SET due_date = now() WHERE id = 1;
+--•	Change a task status
+UPDATE task SET status_id = 3 WHERE id = 5;
+--•	Mark a task as complete.
+UPDATE task SET status_id = (SELECT id FROM status WHERE name = 'Done')WHERE id = 19;
```

Comment:
What do you want to do here,? does the query answers you question?

Reply:
Hi Actually I wrote a sub query to pic status id and replace the status_Id in the task table(it will pick 3 and replace). I can write direct query to update status id because I know id 3 is done.